docs/hardware/backpack/hdzero-goggles.md

```diff
@@ -0,0 +1,47 @@
+---
+template: main.html
+description: How to setup, flash and update your HDZero Goggles ExpressLRS Backpack.
+---
+
+![Hardware-Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/hardware.png)
+
+## Flashing the HDZero Goggles Backpack:
+
+After building the VRx firmware, copy the follwing 4 files into a folder named `ELRS` on the root of the SD card.
+- firmware.bin
+- partitions.bin
+- bootloader.bin
+- boot_app0.bin
+
+Insert the SD card into the goggles and boot the goggles as normal.
+
+The first step is to ensure that you have the backpack enabled in the `Connections` page in the goggles.
+
+![Connections](../../assets/images/hdzero/connections.png)
+
+Scroll down to the `Firmware` page and select the `Update ESP32` option
+
+![Firmware](../../assets/images/hdzero/firmwarepage.png)
+
+The 4 firmware files will be flashed to the ESP32 backpack and you should see the progress bar, with the nanme of the current file being flashed at the start.
+
+![Uploading](../../assets/images/hdzero/uploading.png)
+
+When the upload has completed the page will display the version (or branch) of backpack firmware that is installed next to the update button.
+
+![Complete](../../assets/images/hdzero/uploadcomplete.png)
+
+## Starting Backpack WiFi
+
+To start the WiFi endpoint on the backpack, navigate to the `Connections` page of the goggles UI and select the `Start Backpack WiFi` option. If you have configured your backpack with you home WiFi credentials, it will attempt to connect to that and will fallback to creating an access point, with name `ExpressLRS VRx Backpack` and password `expresslrs`. The web UI will be available as `http://elrs_vrx` or `http://elrs_vrx.local`
```

Comment:
"and will fallback to creating an access point,"
should this be:
"and will fallback to creating an access point if it cannot connect,"

Reply:
done